src/VisualStudio/CSharp/Impl/LanguageService/CSharpHelpContextService.cs

```diff
@@ -375,6 +379,12 @@ private static bool TryGetTextForKeyword(SyntaxToken token, ISyntaxFactsService
                 return true;
             }
 
+            if (token.IsKind(SyntaxKind.UsingKeyword) && token.Parent is UsingStatementSyntax)
+            {
+                text = Keyword("using-statement");
```

Comment:
The docs page this leads to also covers using declarations (`using var x = Foo();`) so it would be good to get a test and support for that at the same time.

Reply:
I updated the PR to now include using declarations.  I only have a simple test for it right now, and it looks like these are disallowed inside places like the expression of an if-statement.  Let me know if I should add additional tests beyond simply testing a local declaration.